xdsl/xdsl_opt_main.py

```diff
@@ -2,11 +2,12 @@
 import sys
 import os
 from io import StringIO
```

Comment:
emptyline

Reply:
What did you mean?

Reply:
To add an empty line between lined 3 and 4